apps/api-documenter/src/documenters/MarkdownDocumenter.ts

```diff
@@ -66,7 +66,6 @@ export class MarkdownDocumenter {
   private readonly _documenterConfig: DocumenterConfig | undefined;
   private readonly _tsdocConfiguration: TSDocConfiguration;
   private readonly _markdownEmitter: CustomMarkdownEmitter;
-  private _outputFolder: string;
```

Comment:
Passing `outputFolder` to every function defeats the benefit of having an object-oriented-class. In this file, the change affects too many members -- I think we need a different solution.